src/datadog/remote_config/remote_config.cpp

```diff
@@ -230,48 +235,46 @@ std::vector<ConfigMetadata> RemoteConfigurationManager::process_response(
         new_config.state = Configuration::State::error;
         new_config.error_message = "Wrong service targeted";
       } else {
-        new_config.state = Configuration::State::acknowledged;
-        new_config.content = parse_dynamic_config(config_json.at("lib_config"));
-
-        auto updated = apply_config(new_config);
-        config_updates.insert(config_updates.end(), updated.begin(),
-                              updated.end());
+        for (const auto& listener : listeners_per_product_[*product]) {
+          // Q: Two listeners on the same product. What should be the behaviour
```

Comment:
this is only a problem if the errors occur on the same config id, not on the same product